.gitlab-ci.yml

```diff
@@ -1,293 +1,266 @@
-  <<: *setup_bundler_env
-  variables:
-    PUPPET_VERSION: '~> 4.10'
-  script:
-    - bundle exec rake spec_clean
-    - bundle exec rake beaker:suites[default]
-
-fips-default:
-  stage: acceptance
-  tags:
-    - beaker
-  <<: *cache_bundler
-  <<: *setup_bundler_env
-  variables:
-    PUPPET_VERSION: '~> 4.10'
-    BEAKER_fips: 'yes'
+pup5.5.7:
+  <<: *pup_5_5_7
+  <<: *acceptance_base
   script:
-    - bundle exec rake spec_clean
-    - bundle exec rake beaker:suites[default]
-
-default-puppet5:
-  stage: acceptance
-  tags:
-    - beaker
-  <<: *cache_bundler
-  <<: *setup_bundler_env
-  variables:
-    PUPPET_VERSION: '~> 5.5'
-    BEAKER_PUPPET_COLLECTION: 'puppet5'
+    - 'bundle exec rake spec_clean'
+    - 'bundle exec rake beaker:suites[default]'
+
+pup5.5.7-fips:
+  <<: *pup_5_5_7
+  <<: *acceptance_base
   script:
-    - bundle exec rake spec_clean
-    - bundle exec rake beaker:suites[default]
-
-fips-default-puppet5:
-  stage: acceptance
-  tags:
-    - beaker
-  <<: *cache_bundler
-  <<: *setup_bundler_env
-  <<: *only_with_SIMP_FULL_MATRIX
-  variables:
-    BEAKER_fips: 'yes'
-    PUPPET_VERSION: '~> 5.5'
-    BEAKER_PUPPET_COLLECTION: 'puppet5'
+    - 'bundle exec rake spec_clean'
+    - 'BEAKER_fips=yes bundle exec rake beaker:suites[default]'
+
+pup5.5.7-ad:
+  <<: *pup_5_5_7
+  <<: *acceptance_base
   script:
-    - bundle exec rake spec_clean
-    - bundle exec rake beaker:suites[default]
-
-default-puppet5-oel:
-  stage: acceptance
-  tags:
-    - beaker
-  <<: *cache_bundler
-  <<: *setup_bundler_env
-  <<: *only_with_SIMP_FULL_MATRIX
-  variables:
-    PUPPET_VERSION: '~> 5.5'
-    BEAKER_PUPPET_COLLECTION: 'puppet5'
+    - 'bundle exec rake spec_clean'
+    - 'bundle exec rake beaker:suites[ad]'
+  retry: 1
+
+pup5.5.7-oel:
+  <<: *pup_5_5_7
+  <<: *acceptance_base
   script:
-    - bundle exec rake spec_clean
-    - bundle exec rake beaker:suites[default,oel]
-
-fips-default-puppet5-oel:
-  stage: acceptance
-  tags:
-    - beaker
-  <<: *cache_bundler
-  <<: *setup_bundler_env
+    - 'bundle exec rake spec_clean'
+    - 'bundle exec rake beaker:suites[default,oel]'
+
+pup-5.5.7-oel-ad:
+  <<: *pup_4_10
+  <<: *acceptance_base
   <<: *only_with_SIMP_FULL_MATRIX
-  variables:
-    BEAKER_fips: 'yes'
-    PUPPET_VERSION: '~> 5.5'
-    BEAKER_PUPPET_COLLECTION: 'puppet5'
   script:
-    - bundle exec rake spec_clean
-    - bundle exec rake beaker:suites[default,oel]
-
-ad:
-  stage: acceptance
-  tags:
-    - beaker
-  <<: *cache_bundler
-  <<: *setup_bundler_env
-  script:
-    - bundle exec rake spec_clean
-    - bundle exec rake beaker:suites[ad]
+    - 'bundle exec rake spec_clean'
+    - 'bundle exec rake beaker:suites[ad,oel]'
   retry: 1
 
-ad-puppet5:
-  stage: acceptance
-  tags:
-    - beaker
-  <<: *cache_bundler
-  <<: *setup_bundler_env
+pup4.10:
+  <<: *pup_4_10
+  <<: *acceptance_base
+  script:
+    - 'bundle exec rake spec_clean'
+    - 'bundle exec rake beaker:suites[default]'
+
+pup4.10-fips:
```

Comment:
Shouldn't this have SIMP_FULL MATRIX only?

Reply:
☑️